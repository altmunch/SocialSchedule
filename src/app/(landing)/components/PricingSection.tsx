'use client';

import { motion } from 'framer-motion';
import { Check, ChevronRight } from 'lucide-react';

interface PricingSectionProps {
  onGetStarted: () => void;
}

export default function PricingSection({ onGetStarted }: PricingSectionProps) {
  const features = [
    'Content Accelerator Optimizing Engine',
    'Precise Automated Posting ("Blitz")',
    'Viral Cycle of Improvements ("Cycle")',
    'Comprehensive Field Research ("Scan")',
    'Retention-Boosting Hashtag & Template Generator',
    'Custom Brand Voice AI',
    'Direct e-commerce platform integration',
    'Priority seller support',
  ];

  return (
    <section className="py-16 md:py-24 bg-white" id="pricing">
      <div className="max-w-7xl mx-auto px-4 sm:px-6 lg:px-8">
        <div className="text-center mb-16">
          <motion.span 
            className="inline-block text-sm font-semibold text-[#FF7F50] uppercase tracking-wider mb-3"
            initial={{ opacity: 0, y: 20 }}
            animate={{ opacity: 1, y: 0 }}
            transition={{ duration: 0.6 }}
          >
            Value & Pricing
          </motion.span>
          
          <motion.h2 
            className="text-3xl md:text-4xl font-bold text-[#333333] mb-6"
            initial={{ opacity: 0, y: 20 }}
            animate={{ opacity: 1, y: 0 }}
            transition={{ duration: 0.6, delay: 0.1 }}
          >
            Exceptional Value for E-Commerce Sellers
          </motion.h2>
          
          <motion.p 
            className="text-xl text-[#444444] max-w-3xl mx-auto"
            initial={{ opacity: 0, y: 20 }}
            animate={{ opacity: 1, y: 0 }}
            transition={{ duration: 0.6, delay: 0.1 }}
          >
            $3300+ value for less than 10% of the cost. All features included. No hidden fees.
          </motion.p>
        </div>

        <motion.div
          className="bg-white border-2 border-[#007BFF] rounded-xl overflow-hidden max-w-xl mx-auto shadow-xl"
          initial={{ opacity: 0, y: 20 }}
          animate={{ opacity: 1, y: 0 }}
          transition={{ duration: 0.6, delay: 0.2 }}
        >
          {/* Popular badge */}
          <div className="bg-[#007BFF] text-white text-center py-2 font-semibold">
            RECOMMENDED FOR SELLERS
          </div>
          
          <div className="p-8 md:p-10">
            <div className="flex items-baseline justify-center mb-6">
              <h3 className="text-4xl md:text-5xl font-bold text-[#333333]">$297</h3>
              <span className="text-[#666666] ml-2">/month</span>
            </div>
            
            <p className="text-center text-[#666666] mb-6">
              <span className="line-through">$497</span> <span className="text-[#28A745] font-semibold">Save 40%</span>
            </p>
            
            <div className="bg-blue-50 rounded-lg p-4 text-center mb-8 border border-blue-100">
              <p className="text-[#333333]">Value of hiring a dedicated specialist: <span className="text-[#28A745] font-semibold">$3,300/mo</span></p>
              <p className="text-[#666666] text-sm">You save over $3,000 per month</p>
            </div>
            
            <motion.button
              onClick={onGetStarted}
              className="w-full bg-[#007BFF] hover:bg-[#0070E0] text-white py-4 rounded-md font-semibold text-lg shadow-md transition-all mb-8"
              whileHover={{ scale: 1.02 }}
              whileTap={{ scale: 0.98 }}
            >
              Start Your 14-Day Trial <ChevronRight className="ml-1 h-5 w-5 inline" />
            </motion.button>
            
            <div className="text-center text-sm text-[#666666] mb-8">
              Cancel anytime during the trial period and you won't be charged
            </div>
            
            <div className="space-y-4">
              {features.map((feature, index) => (
                <motion.div 
                  key={index} 
                  className="flex items-center"
                  initial={{ opacity: 0, x: -10 }}
                  animate={{ opacity: 1, x: 0 }}
                  transition={{ 
                    duration: 0.3, 
                    delay: 0.2 + (index * 0.05),
                    ease: "easeOut"
                  }}
                >
                  <Check className="h-5 w-5 text-[#28A745] mr-3 flex-shrink-0" />
                  <span className="text-[#444444]">{feature}</span>
                </motion.div>
              ))}
            </div>
          </div>
        </motion.div>
      </div>
    </section>
  );
}
