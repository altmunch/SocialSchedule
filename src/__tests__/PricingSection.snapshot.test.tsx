import { render } from '@testing-library/react';
import PricingSection from '@/app/landing/components/PricingSection';

describe('PricingSection', () => {
  it('matches snapshot', () => {
    const { container } = render(<PricingSection onGetStarted={() => {}} />);
    expect(container).toMatchSnapshot();
  });
}); 