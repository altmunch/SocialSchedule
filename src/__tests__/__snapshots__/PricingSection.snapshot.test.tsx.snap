// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PricingSection matches snapshot 1`] = `
<div>
  <section
    class="py-16 md:py-24 bg-gradient-to-b from-black to-[#0A0A0A] text-white"
  >
    <div
      class="max-w-7xl mx-auto px-6 sm:px-12 lg:px-16"
    >
      <div
        class="text-center mb-16"
        initial="[object Object]"
        transition="[object Object]"
        viewport="[object Object]"
        whileinview="[object Object]"
      >
        <h2
          class="text-4xl md:text-6xl font-bold mb-6"
        >
          Choose Your 
          <span
            class="text-[#5afcc0]"
          >
            Revenue
          </span>
           Plan
        </h2>
        <p
          class="text-xl text-white/80 max-w-3xl mx-auto"
        >
          Every plan is designed to pay for itself within the first month through increased sales and conversions.
        </p>
      </div>
      <div
        class="text-center mb-16"
      >
        <span
          class="inline-block text-sm font-semibold text-[#5afcc0] uppercase tracking-wider mb-3"
          initial="[object Object]"
          transition="[object Object]"
          viewport="[object Object]"
          whileinview="[object Object]"
        >
          Simple, Transparent Pricing
        </span>
        <h2
          class="text-3xl md:text-4xl font-bold text-white mb-6"
          initial="[object Object]"
          transition="[object Object]"
          viewport="[object Object]"
          whileinview="[object Object]"
        >
          Scale Your Content and 
          <span
            class="text-[#8D5AFF]"
          >
            Maximize Sales
          </span>
        </h2>
        <p
          class="text-xl text-white/80 max-w-3xl mx-auto mb-10"
          initial="[object Object]"
          transition="[object Object]"
          viewport="[object Object]"
          whileinview="[object Object]"
        >
          Select the plan that best fits your needs. All plans include our core AI-powered features.
        </p>
        <div
          class="inline-flex items-center bg-black/30 p-1 rounded-full border border-white/10 mb-12 backdrop-blur-sm"
          initial="[object Object]"
          transition="[object Object]"
          viewport="[object Object]"
          whileinview="[object Object]"
        >
          <button
            class="px-6 py-2 rounded-full text-sm font-medium transition-all bg-[#8D5AFF] text-white shadow-lg"
          >
            Annual
            <span
              class="ml-1 text-xs bg-[#5afcc0] text-black px-2 py-0.5 rounded-full"
            >
              Save 20%
            </span>
          </button>
          <button
            class="px-6 py-2 rounded-full text-sm font-medium transition-all text-white/60 hover:text-white"
          >
            Monthly
          </button>
        </div>
        <div
          class="grid md:grid-cols-3 gap-6 xl:gap-10"
        >
          <div
            class="relative rounded-2xl overflow-hidden backdrop-blur-sm border border-white/10 bg-black/40"
            initial="[object Object]"
            transition="[object Object]"
            viewport="[object Object]"
            whileinview="[object Object]"
          >
            <div
              class="p-8 relative"
            >
              <h3
                class="text-2xl font-bold text-white mb-2"
              >
                Lite
              </h3>
              <p
                class="text-white/60 mb-6 min-h-[40px]"
              >
                $20/month
              </p>
              <div
                class="flex items-baseline mb-6"
              >
                <span
                  class="text-4xl font-bold text-white"
                >
                  $
                  240
                </span>
                <span
                  class="text-white/60 ml-2"
                >
                  /year
                </span>
              </div>
              <div
                class="bg-[#5afcc0]/10 rounded-lg p-4 mb-6 border border-[#5afcc0]/20"
              >
                <div
                  class="flex items-start"
                >
                  <svg
                    class="h-5 w-5 text-[#5afcc0] mr-2 mt-0.5 flex-shrink-0"
                    data-testid="Sparkles-icon"
                  />
                  <div>
                    <p
                      class="text-[#5afcc0] font-medium"
                    >
                      Limited Time Bonuses
                    </p>
                    <p
                      class="text-sm text-white/70"
                    >
                      Template Generator & Hook Creator included ($899 value)
                    </p>
                  </div>
                </div>
              </div>
              <button
                class="w-full group relative py-3 px-6 rounded-lg font-semibold text-white transition-all duration-300 overflow-hidden mb-8 bg-[#8D5AFF] hover:bg-[#8D5AFF]/90"
              >
                <span
                  class="relative z-10"
                >
                  Select Plan
                </span>
                <svg
                  class="ml-2 h-5 w-5 inline transition-transform group-hover:translate-x-1"
                  data-testid="ChevronRight-icon"
                />
              </button>
              <div
                class="flex items-center justify-center text-sm text-white/60 mb-6"
              >
                <svg
                  class="h-4 w-4 mr-2 text-[#5afcc0]"
                  data-testid="Shield-icon"
                />
                <span>
                  10-day results guarantee
                </span>
              </div>
              <div
                class="space-y-3"
              >
                <p
                  class="text-sm font-medium text-white border-b border-white/10 pb-2 mb-3"
                >
                  FEATURES
                </p>
                <div
                  class="flex items-start"
                  initial="[object Object]"
                  transition="[object Object]"
                  viewport="[object Object]"
                  whileinview="[object Object]"
                >
                  <div
                    class="w-5 h-5 rounded-full bg-[#5afcc0]/10 flex items-center justify-center flex-shrink-0 mr-3 mt-0.5"
                  >
                    <svg
                      class="h-3 w-3 text-[#5afcc0]"
                      data-testid="Check-icon"
                    />
                  </div>
                  <span
                    class="text-white/80 text-sm"
                  >
                    Viral Blitz Cycle Framework (15 uses)
                  </span>
                </div>
                <div
                  class="flex items-start"
                  initial="[object Object]"
                  transition="[object Object]"
                  viewport="[object Object]"
                  whileinview="[object Object]"
                >
                  <div
                    class="w-5 h-5 rounded-full bg-[#5afcc0]/10 flex items-center justify-center flex-shrink-0 mr-3 mt-0.5"
                  >
                    <svg
                      class="h-3 w-3 text-[#5afcc0]"
                      data-testid="Check-icon"
                    />
                  </div>
                  <span
                    class="text-white/80 text-sm"
                  >
                    Idea Generator Framework (15 uses)
                  </span>
                </div>
                <div
                  class="flex items-start"
                  initial="[object Object]"
                  transition="[object Object]"
                  viewport="[object Object]"
                  whileinview="[object Object]"
                >
                  <div
                    class="w-5 h-5 rounded-full bg-[#5afcc0]/10 flex items-center justify-center flex-shrink-0 mr-3 mt-0.5"
                  >
                    <svg
                      class="h-3 w-3 text-[#5afcc0]"
                      data-testid="Check-icon"
                    />
                  </div>
                  <span
                    class="text-white/80 text-sm"
                  >
                    15 autoposts/month
                  </span>
                </div>
                <div
                  class="flex items-start"
                  initial="[object Object]"
                  transition="[object Object]"
                  viewport="[object Object]"
                  whileinview="[object Object]"
                >
                  <div
                    class="w-5 h-5 rounded-full bg-[#5afcc0]/10 flex items-center justify-center flex-shrink-0 mr-3 mt-0.5"
                  >
                    <svg
                      class="h-3 w-3 text-[#5afcc0]"
                      data-testid="Check-icon"
                    />
                  </div>
                  <span
                    class="text-white/80 text-sm"
                  >
                    Basic analytics (no e-commerce)
                  </span>
                </div>
              </div>
            </div>
          </div>
          <div
            class="relative rounded-2xl overflow-hidden backdrop-blur-sm border-2 border-[#8D5AFF]/50 bg-gradient-to-br from-black to-[#1A1A1A]"
            initial="[object Object]"
            transition="[object Object]"
            viewport="[object Object]"
            whileinview="[object Object]"
          >
            <div
              class="bg-gradient-to-r from-[#8D5AFF] to-[#5afcc0] text-white text-center py-3 font-semibold text-sm tracking-wider"
            >
              MOST POPULAR
            </div>
            <div
              class="p-8 relative"
            >
              <div
                class="absolute -top-20 -right-20 w-64 h-64 bg-[#8D5AFF]/20 rounded-full filter blur-3xl -z-10"
              />
              <h3
                class="text-2xl font-bold text-white mb-2"
              >
                Pro
              </h3>
              <p
                class="text-white/60 mb-6 min-h-[40px]"
              >
                $70/month
              </p>
              <div
                class="flex items-baseline mb-6"
              >
                <span
                  class="text-4xl font-bold text-white"
                >
                  $
                  840
                </span>
                <span
                  class="text-white/60 ml-2"
                >
                  /year
                </span>
              </div>
              <div
                class="bg-[#5afcc0]/10 rounded-lg p-4 mb-6 border border-[#5afcc0]/20"
              >
                <div
                  class="flex items-start"
                >
                  <svg
                    class="h-5 w-5 text-[#5afcc0] mr-2 mt-0.5 flex-shrink-0"
                    data-testid="Sparkles-icon"
                  />
                  <div>
                    <p
                      class="text-[#5afcc0] font-medium"
                    >
                      Limited Time Bonuses
                    </p>
                    <p
                      class="text-sm text-white/70"
                    >
                      Template Generator & Hook Creator included ($899 value)
                    </p>
                  </div>
                </div>
              </div>
              <button
                class="w-full group relative py-3 px-6 rounded-lg font-semibold text-white transition-all duration-300 overflow-hidden mb-8 bg-gradient-to-r from-[#8D5AFF] to-[#5afcc0] hover:shadow-lg hover:shadow-[#8D5AFF]/20"
              >
                <span
                  class="relative z-10"
                >
                  Get Started
                </span>
                <svg
                  class="ml-2 h-5 w-5 inline transition-transform group-hover:translate-x-1"
                  data-testid="ChevronRight-icon"
                />
              </button>
              <div
                class="flex items-center justify-center text-sm text-white/60 mb-6"
              >
                <svg
                  class="h-4 w-4 mr-2 text-[#5afcc0]"
                  data-testid="Shield-icon"
                />
                <span>
                  10-day results guarantee
                </span>
              </div>
              <div
                class="space-y-3"
              >
                <p
                  class="text-sm font-medium text-white border-b border-white/10 pb-2 mb-3"
                >
                  FEATURES
                </p>
                <div
                  class="flex items-start"
                  initial="[object Object]"
                  transition="[object Object]"
                  viewport="[object Object]"
                  whileinview="[object Object]"
                >
                  <div
                    class="w-5 h-5 rounded-full bg-[#5afcc0]/10 flex items-center justify-center flex-shrink-0 mr-3 mt-0.5"
                  >
                    <svg
                      class="h-3 w-3 text-[#5afcc0]"
                      data-testid="Check-icon"
                    />
                  </div>
                  <span
                    class="text-white/80 text-sm"
                  >
                    Viral Blitz Cycle Framework (unlimited)
                  </span>
                </div>
                <div
                  class="flex items-start"
                  initial="[object Object]"
                  transition="[object Object]"
                  viewport="[object Object]"
                  whileinview="[object Object]"
                >
                  <div
                    class="w-5 h-5 rounded-full bg-[#5afcc0]/10 flex items-center justify-center flex-shrink-0 mr-3 mt-0.5"
                  >
                    <svg
                      class="h-3 w-3 text-[#5afcc0]"
                      data-testid="Check-icon"
                    />
                  </div>
                  <span
                    class="text-white/80 text-sm"
                  >
                    Idea Generator Framework (unlimited)
                  </span>
                </div>
                <div
                  class="flex items-start"
                  initial="[object Object]"
                  transition="[object Object]"
                  viewport="[object Object]"
                  whileinview="[object Object]"
                >
                  <div
                    class="w-5 h-5 rounded-full bg-[#5afcc0]/10 flex items-center justify-center flex-shrink-0 mr-3 mt-0.5"
                  >
                    <svg
                      class="h-3 w-3 text-[#5afcc0]"
                      data-testid="Check-icon"
                    />
                  </div>
                  <span
                    class="text-white/80 text-sm"
                  >
                    Unlimited posts
                  </span>
                </div>
                <div
                  class="flex items-start"
                  initial="[object Object]"
                  transition="[object Object]"
                  viewport="[object Object]"
                  whileinview="[object Object]"
                >
                  <div
                    class="w-5 h-5 rounded-full bg-[#5afcc0]/10 flex items-center justify-center flex-shrink-0 mr-3 mt-0.5"
                  >
                    <svg
                      class="h-3 w-3 text-[#5afcc0]"
                      data-testid="Check-icon"
                    />
                  </div>
                  <span
                    class="text-white/80 text-sm"
                  >
                    Multiple account sets
                  </span>
                </div>
                <div
                  class="flex items-start"
                  initial="[object Object]"
                  transition="[object Object]"
                  viewport="[object Object]"
                  whileinview="[object Object]"
                >
                  <div
                    class="w-5 h-5 rounded-full bg-[#5afcc0]/10 flex items-center justify-center flex-shrink-0 mr-3 mt-0.5"
                  >
                    <svg
                      class="h-3 w-3 text-[#5afcc0]"
                      data-testid="Check-icon"
                    />
                  </div>
                  <span
                    class="text-white/80 text-sm"
                  >
                    E-commerce integration
                  </span>
                </div>
                <div
                  class="flex items-start"
                  initial="[object Object]"
                  transition="[object Object]"
                  viewport="[object Object]"
                  whileinview="[object Object]"
                >
                  <div
                    class="w-5 h-5 rounded-full bg-[#5afcc0]/10 flex items-center justify-center flex-shrink-0 mr-3 mt-0.5"
                  >
                    <svg
                      class="h-3 w-3 text-[#5afcc0]"
                      data-testid="Check-icon"
                    />
                  </div>
                  <span
                    class="text-white/80 text-sm"
                  >
                    Advanced analytics & reporting
                  </span>
                </div>
              </div>
            </div>
            <div
              class="absolute -bottom-20 -left-20 w-64 h-64 bg-[#5afcc0]/10 rounded-full filter blur-3xl -z-10"
            />
          </div>
          <div
            class="relative rounded-2xl overflow-hidden backdrop-blur-sm border border-white/10 bg-black/40"
            initial="[object Object]"
            transition="[object Object]"
            viewport="[object Object]"
            whileinview="[object Object]"
          >
            <div
              class="p-8 relative"
            >
              <h3
                class="text-2xl font-bold text-white mb-2"
              >
                Team
              </h3>
              <p
                class="text-white/60 mb-6 min-h-[40px]"
              >
                $500/month
              </p>
              <div
                class="flex items-baseline mb-6"
              >
                <span
                  class="text-4xl font-bold text-white"
                >
                  $
                  6000
                </span>
                <span
                  class="text-white/60 ml-2"
                >
                  /year
                </span>
              </div>
              <div
                class="bg-[#5afcc0]/10 rounded-lg p-4 mb-6 border border-[#5afcc0]/20"
              >
                <div
                  class="flex items-start"
                >
                  <svg
                    class="h-5 w-5 text-[#5afcc0] mr-2 mt-0.5 flex-shrink-0"
                    data-testid="Sparkles-icon"
                  />
                  <div>
                    <p
                      class="text-[#5afcc0] font-medium"
                    >
                      Limited Time Bonuses
                    </p>
                    <p
                      class="text-sm text-white/70"
                    >
                      Template Generator & Hook Creator included ($899 value)
                    </p>
                  </div>
                </div>
              </div>
              <button
                class="w-full group relative py-3 px-6 rounded-lg font-semibold text-white transition-all duration-300 overflow-hidden mb-8 bg-[#8D5AFF] hover:bg-[#8D5AFF]/90"
              >
                <span
                  class="relative z-10"
                >
                  Select Plan
                </span>
                <svg
                  class="ml-2 h-5 w-5 inline transition-transform group-hover:translate-x-1"
                  data-testid="ChevronRight-icon"
                />
              </button>
              <div
                class="flex items-center justify-center text-sm text-white/60 mb-6"
              >
                <svg
                  class="h-4 w-4 mr-2 text-[#5afcc0]"
                  data-testid="Shield-icon"
                />
                <span>
                  10-day results guarantee
                </span>
              </div>
              <div
                class="space-y-3"
              >
                <p
                  class="text-sm font-medium text-white border-b border-white/10 pb-2 mb-3"
                >
                  FEATURES
                </p>
                <div
                  class="flex items-start"
                  initial="[object Object]"
                  transition="[object Object]"
                  viewport="[object Object]"
                  whileinview="[object Object]"
                >
                  <div
                    class="w-5 h-5 rounded-full bg-[#5afcc0]/10 flex items-center justify-center flex-shrink-0 mr-3 mt-0.5"
                  >
                    <svg
                      class="h-3 w-3 text-[#5afcc0]"
                      data-testid="Check-icon"
                    />
                  </div>
                  <span
                    class="text-white/80 text-sm"
                  >
                    Everything in Pro
                  </span>
                </div>
                <div
                  class="flex items-start"
                  initial="[object Object]"
                  transition="[object Object]"
                  viewport="[object Object]"
                  whileinview="[object Object]"
                >
                  <div
                    class="w-5 h-5 rounded-full bg-[#5afcc0]/10 flex items-center justify-center flex-shrink-0 mr-3 mt-0.5"
                  >
                    <svg
                      class="h-3 w-3 text-[#5afcc0]"
                      data-testid="Check-icon"
                    />
                  </div>
                  <span
                    class="text-white/80 text-sm"
                  >
                    Team dashboard access
                  </span>
                </div>
                <div
                  class="flex items-start"
                  initial="[object Object]"
                  transition="[object Object]"
                  viewport="[object Object]"
                  whileinview="[object Object]"
                >
                  <div
                    class="w-5 h-5 rounded-full bg-[#5afcc0]/10 flex items-center justify-center flex-shrink-0 mr-3 mt-0.5"
                  >
                    <svg
                      class="h-3 w-3 text-[#5afcc0]"
                      data-testid="Check-icon"
                    />
                  </div>
                  <span
                    class="text-white/80 text-sm"
                  >
                    Manage unlimited accounts
                  </span>
                </div>
                <div
                  class="flex items-start"
                  initial="[object Object]"
                  transition="[object Object]"
                  viewport="[object Object]"
                  whileinview="[object Object]"
                >
                  <div
                    class="w-5 h-5 rounded-full bg-[#5afcc0]/10 flex items-center justify-center flex-shrink-0 mr-3 mt-0.5"
                  >
                    <svg
                      class="h-3 w-3 text-[#5afcc0]"
                      data-testid="Check-icon"
                    />
                  </div>
                  <span
                    class="text-white/80 text-sm"
                  >
                    Brand Voice AI (for consistency)
                  </span>
                </div>
                <div
                  class="flex items-start"
                  initial="[object Object]"
                  transition="[object Object]"
                  viewport="[object Object]"
                  whileinview="[object Object]"
                >
                  <div
                    class="w-5 h-5 rounded-full bg-[#5afcc0]/10 flex items-center justify-center flex-shrink-0 mr-3 mt-0.5"
                  >
                    <svg
                      class="h-3 w-3 text-[#5afcc0]"
                      data-testid="Check-icon"
                    />
                  </div>
                  <span
                    class="text-white/80 text-sm"
                  >
                    Team collaboration mode
                  </span>
                </div>
                <div
                  class="flex items-start"
                  initial="[object Object]"
                  transition="[object Object]"
                  viewport="[object Object]"
                  whileinview="[object Object]"
                >
                  <div
                    class="w-5 h-5 rounded-full bg-[#5afcc0]/10 flex items-center justify-center flex-shrink-0 mr-3 mt-0.5"
                  >
                    <svg
                      class="h-3 w-3 text-[#5afcc0]"
                      data-testid="Check-icon"
                    />
                  </div>
                  <span
                    class="text-white/80 text-sm"
                  >
                    Priority support
                  </span>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </section>
</div>
`;
